controllers/login.js

```diff
@@ -133,13 +133,9 @@ router.all('/login/superhero/', (req, res, next) => {
 	handleLoginFailed(req, res).catch(next);
 });
 
-const ssoSchoolData = (req, systemId) => api(req).get('/schools/', {
-	qs: {
-		systems: systemId,
-	},
-}).then((schools) => {
-	if (schools.data.length > 0) {
-		return schools.data[0];
+const ssoSchoolData = (req, systemId) => api(req).get('/schoolsList').then((schools) => {
+	if (schools.length > 0) {
+		return schools.find((school) => school.systems.find((system) => system._id === systemId));
```

Comment:
Can we pass the requested systemId as optional query to backend to get the reduced list directly?

Reply:
My idea was to have an endpoint which is as simple as possible, with no params, no extra api calls needed.
This is only used from login, and having the cache (increased form 1 to 30 minutes now), it should be performant.